genestack_client/settings/genestack_config.py

```diff
@@ -194,5 +168,31 @@ def save(self):
         with open(config_path, 'w') as f:
             document.writexml(f, indent='', addindent='    ', newl='\n')
 
+    def store_secure_value(self, alias, secret_item, key, document, user_element, element_name):
```

Comment:
`store_value_securely`

Reply:
1. This need documenting.
2. I would expect arg order and naming like this: `(self, user_alias, key, value, document, user_element, element_name)`.
3. Ideally I would like to have this function accept only `(self, user_alias, key, value)` (only info required for storing in keychain) and returning XML (sub)element (if any) to write into XML config.